contrib/test_jenkins.sh

```diff
@@ -1014,6 +1015,63 @@ test_memtrack() {
 	UCX_MEMTRACK_DEST=stdout UCX_HANDLE_ERRORS=none UCX_MEMTRACK_LIMIT=412MB ./test/apps/test_memtrack_limit |& grep -C 100 'reached'
 }
 
+test_namespace() {
```

Comment:
why is it part of test_jenkins.sh and not a separate script?

Reply:
now moved to `test_namespace.sh`